cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/core/mongo/impl/PublicationDaoImpl.java

```diff
@@ -68,47 +72,83 @@
         return results;
     }
 
+    @Override
+    public List<PublicationImpl> getPublications(Pageable pageable, PublicationImpl pub, boolean init) {
```

Comment:
what is `init` for? also, javadoc

Reply:
I might be getting rid of the `init` variable in the new implementation. Added a javadoc for this method.